.gitignore

```diff
@@ -4,3 +4,4 @@ dist/**/*
 **/*.egg-info/**/*
 .sass-cache/*
 /build
+/_readthedocs/
```

Comment:
I don't think this should be added.

Reply:
Removed in [a84f63f](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/139/commits/a84f63fcfe0e77d60591a27d2e882c3bd812e212)